h/services/group.py

```diff
@@ -59,14 +55,42 @@ def create_open_group(self, name, userid, origins, description=None):
         :param name: the human-readable name of the group
         :param userid: the userid of the group creator
         :param description: the description of the group
+        :param type_flags: the type of this group
+        :param origins: request header indicating where a request originates from
+        :param add_creator_as_member: if the group creator should be added as a member
```

Comment:
Same here, `type_flags` and `add_creator_as_member` aren't param of this method

Reply:
Thanks for adding the `origins` param to the docstring, but the docstring for it is wrong - it's actually the list of origins (e.g. `["https://example.com", "https://biopub.hypothes.is"]`) that the group will be scoped to